base/applications/mscutils/eventvwr/lang/en-US.rc

```diff
@@ -140,12 +140,12 @@ BEGIN
     LTEXT "KB", IDC_STATIC, 130, 122, 20, 8
     LTEXT "When maximum log size is reached:", IDC_STATIC, 17, 140, 219, 8
     CONTROL "&Overwrite events as needed", IDC_OVERWRITE_AS_NEEDED, "Button", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 17, 155, 219, 10
-    CONTROL "O&verwrite events older than", IDC_OVERWRITE_OLDER_THAN, "Button", BS_AUTORADIOBUTTON | WS_GROUP | WS_TABSTOP, 17, 168, 102, 10
+    CONTROL "O&verwrite events older than", IDC_OVERWRITE_OLDER_THAN, "Button", BS_AUTORADIOBUTTON | WS_TABSTOP, 17, 168, 102, 10
+    CONTROL "Do &not overwrite events\n(clear log manually)", IDC_NO_OVERWRITE, "Button", BS_AUTORADIOBUTTON | BS_MULTILINE | WS_TABSTOP, 17, 180, 143, 20
     EDITTEXT IDC_EDIT_EVENTS_AGE, 122, 165, 35, 14, ES_LEFT | ES_NUMBER | WS_GROUP
     CONTROL "", IDC_UPDOWN_EVENTS_AGE, UPDOWN_CLASS, UDS_SETBUDDYINT | UDS_ALIGNRIGHT |
-            UDS_AUTOBUDDY | UDS_ARROWKEYS | WS_GROUP, 157, 165, 10, 14
+            UDS_AUTOBUDDY | UDS_ARROWKEYS, 157, 165, 10, 14
     LTEXT "days", IDC_STATIC, 162, 168, 20, 8
-    CONTROL "Do &not overwrite events\n(clear log manually)", IDC_NO_OVERWRITE, "Button", BS_AUTORADIOBUTTON | BS_MULTILINE | WS_GROUP | WS_TABSTOP, 17, 180, 143, 20
     PUSHBUTTON "&Restore Defaults", IDC_RESTOREDEFAULTS, 166, 183, 70, 14
```

Comment:
(Comment for all languages): The button doesn't need a WS_GROUP however? How would this change the navigation behaviour?

Reply:
If you are talking about the restore defaults button previously it was possible to use the arrow keys to put focus on the Restore Defaults button when a radiobutton was focused however picking between the radiobuttons was not possible, currently with the arrow keys its possible to pick between the radiobuttons but a TAB is needed to select the Restore Defaults button, 

Not sure how much the video helps but left is before, right is after

https://github.com/reactos/reactos/assets/36992813/ee9162c7-f224-40f7-b511-9dc038658abe